src/components/input/bl-input.stories.mdx

```diff
@@ -86,6 +86,27 @@ import { extraPadding } from '../../utilities/chromatic-decorators';
 export const BlButton = args => html`
   <bl-button>Check</bl-button>`
 
+export const SpinnerIcon = args => html`
+  <style>
+    .spinner {
+      animation: spin 1s linear infinite;
+      color: var(--bl-color-primary);
+    }
+
+    @keyframes spin {
+      0% {
+        transform: rotate(0deg);
+      }
+
+      100% {
+        transform: rotate(360deg);
+      }
+    }
+  </style>
```

Comment:
Putting style tag inside `bl-input` would misguide the people. I think it's better to put style tag outside.